go.mod

```diff
@@ -214,3 +214,8 @@ require (
 	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace (
+	antrea.io/libOpenflow v0.10.2 => github.com/wenyingd/libOpenflow v0.0.0-20230619081321-27a81f0f224c
+	antrea.io/ofnet v0.7.2 => github.com/wenyingd/ofnet v0.0.0-20230619081658-087e990a25d7
```

Comment:
Why to use personal repo? no plan to bump up libOpenflow and ofnet?

Reply:
The dependent two patches are not merged yet, here I just want to trigger test to ensure everything works as expected.